examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt

```diff
@@ -309,4 +317,31 @@ class FasterRouteActivity : AppCompatActivity(), OnMapReadyCallback {
         override fun onFailure(exception: Exception) {
         }
     }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        mapView.onSaveInstanceState(outState)
+        // This is not the most efficient way to preserve the route on a device rotation.
+        // This is here to demonstrate that this event needs to be handled in order to
+        // redraw the route line after a rotation.
+        directionRoute?.let {
+            outState.putString(Utils.PRIMARY_ROUTE_BUNDLE_KEY, it.toJson())
```

Comment:
the same as above(1 Mb limit)

btw if we have `Utils.getRouteFromBundle(Bundle?)` seems that would be great to have  `Utils.saveRouteTo(Bundle?)` and hide `PRIMARY_ROUTE_BUNDLE_KEY `